### PR TITLE
Adds relatesTo and minor improvements to relationships in cypherbuilder

### PR DESCRIPTION
**Pull request by angrykoala**
# Description
This PR adds the `relatedTo` method to NodeRef and minor improvements on how cypher is generated

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Made some changes to the GraphQL Toolbox? Or made some changes to the core library perhaps?

Check out your changes at
 http://neo4j-graphql-toolbox-stage-2104.surge.sh (=> for local development; use for instance [Neo4j Desktop](https://neo4j.com/download/))
or
 https://neo4j-graphql-toolbox-stage-2104.surge.sh (=> use to connect to an [Aura Free](https://neo4j.com/cloud/aura-free/) dbms)

File: packages/graphql/src/translate/cypher-builder/Pattern.ts [hunk 1]
```diff
@@ -157,4 +157,15 @@ export class Pattern<T extends MatchableElement = any> implements CypherCompilab
...
         return padLeft(stringifyObject(paramValues));
     }
+
+    private getNodeLabelsString(node: NodeRef): string {
+        const escapedLabels = node.labels.map(escapeLabel);
+        if (escapedLabels.length === 0) return "";
+        return `:${escapedLabels.join(":")}`;
+    }
+
+    private getRelationshipLabelsString(relationship: RelationshipRef): string {
```
Comment (tbwiss) [inline packages/graphql/src/translate/cypher-builder/Pattern.ts:167]: question (non-blocking): Do you also think `getRelationshipTypesString` is more accurate in this case?
```suggestion
    private getRelationshipTypesString(relationship: RelationshipRef): string {
```

File: packages/graphql/src/translate/cypher-builder/variables/RelationshipRef.test.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as CypherBuilder from "../CypherBuilder";
+import { TestClause } from "../utils/TestClause";
+
+describe("RelationshipRef", () => {
+    it("Create Relationshipt from node", () => {
```
Comment (tbwiss) [inline packages/graphql/src/translate/cypher-builder/variables/RelationshipRef.test.ts:24]: A typo must have sneaked itself into the PR
```suggestion
    it("Create relationship from node", () => {
```

File: packages/graphql/src/translate/cypher-builder/Pattern.ts [hunk 3]
```diff
@@ -157,4 +157,15 @@ export class Pattern<T extends MatchableElement = any> implements CypherCompilab
...
+        // TODO: escapeLabel
```
Comment (tbwiss) [inline packages/graphql/src/translate/cypher-builder/Pattern.ts:168]: question (non-blocking): for my understanding, is the escaping of node labels any different from the escaping of rel types?

**Review by tbwiss — APPROVED** [review]
LGTM! Just have a few minor comments.

File: packages/graphql/src/translate/cypher-builder/Pattern.ts (hunk 1 above)
Comment (angrykoala) [reply packages/graphql/src/translate/cypher-builder/Pattern.ts:167]: 100% agree

File: packages/graphql/src/translate/cypher-builder/Pattern.ts (hunk 3 above)
Comment (angrykoala) [reply packages/graphql/src/translate/cypher-builder/Pattern.ts:168]: Nope, but due to it being not a high priority and a breaking change we are not escaping them until next major release

**Pull request merged**